OsmAnd-java/src/main/java/net/osmand/binary/ObfConstants.java

```diff
@@ -23,7 +23,7 @@ public class ObfConstants {
 	
 	
 	public static final long RELATION_BIT = 1L << (ObfConstants.SHIFT_MULTIPOLYGON_IDS - 1); // 1L << 42
-	public static final long PROPAGATE_NODE_BIT = 1L << (ObfConstants.SHIFT_PROPAGATED_NODE_IDS  - 1); // 1L << 41
+	public static final long PROPAGATE_NODE_BIT = 1L << (ObfConstants.SHIFT_PROPAGATED_NODE_IDS  - 1); // 1L << 49
```

Comment:
how does it affect backward compatibility?

Reply:
Comment change only, not real number